packages/atomic/src/utils/props-utils.ts

```diff
@@ -1,12 +1,44 @@
 import {isArray} from '@coveo/bueno';
 import {ComponentInterface, getElement} from '@stencil/core';
+import {ReactiveElement} from 'lit';
 import {camelToKebab, kebabToCamel} from './utils';
 
 interface MapPropOptions {
   attributePrefix?: string;
   splitValues?: boolean;
 }
 
+export function mapProperty<Element extends ReactiveElement>(
+  options?: MapPropOptions
+) {
+  return <
+    Instance extends Element & Record<string, unknown>,
+    K extends keyof Instance,
+  >(
+    proto: ReactiveElement,
+    propertyKey: K
+  ) => {
+    const ctor = proto.constructor as typeof ReactiveElement;
+    ctor.addInitializer((instance) => {
+      const props = {};
+      const prefix =
+        options?.attributePrefix || camelToKebab(propertyKey as string);
```

Comment:
```suggestion
        options?.attributePrefix || camelToKebab(propertyKey.toString());
```

would this work instead of casting?